src/modules/test/tests/testrecipes/sample.json

```diff
@@ -0,0 +1,12 @@
+[
```

Comment:
The folder can be named "recipes" instead of "testrecipes". 

Also tests can be excluded.

Right now we have things like this:

src/modules/test/tests/
src/modules/test/testrecipes/

etc.

Try to avoid "test/test"